daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/EngineConfig.scala

```diff
@@ -97,4 +40,23 @@ object EngineConfig {
     outputTransactionVersions = VersionRange(TV("10"), TV("10"))
   )
 
+  // development configuration, should not be used in PROD.
+  // accept all language and transaction versions supported by SDK_1_x plus development versions.
+  val Dev: EngineConfig = toDev(Stable)
+
+  // Legacy configuration, to be used by sandbox classic only
+  @deprecated("Sandbox_Classic_Stable is to be used by sandbox classic only", since = "1.5.0")
+  val Sandbox_Classic_Stable: EngineConfig =
+    Stable.copy(
+      languageVersions = Stable.languageVersions.copy(min = LV(LV.Major.V0, LV.Minor.Stable(""))),
```

Comment:
This seems to have changed from v1.1 to v0. Was this intentional?

Reply:
Yes. But, it does not really matter as we does not really support V0.
I should probably remove it.